pychromecast/controllers/media.py

```diff
@@ -67,6 +69,16 @@ def __init__(self):
         self.media_metadata = {}
         self.subtitle_tracks = {}
         self.current_subtitle_tracks = []
+        self.last_updated = None
+
+    def get_seek_time(self):
+        """ Returns current seek time of media in seconds """
+        if self.player_state == MEDIA_PLAYER_STATE_PLAYING:
+            # Add time since last update
+            return (self.current_time +
+                    datetime.utcnow() - self.last_updated).total_seconds()
```

Comment:
I'm a bit surprised here, shouldn't you put parentheses around the `now - last_updated` part?

Reply:
Technically it doesn't matter, and the line break provides some clarity... I did take them out because it made the line break indentation look weird. Want them back in?

Reply:
This is my first time really using python (I'm usually a JS guy) so sorry about any bad practices I may be making!

Reply:
(First pull request too actually... hopefully many more ahead!)

Reply:
let's keep them around it, code is more often read than that it is written.

Reply:
Welcome to open source, you're doing great work 🎉